src/structures/session.ts

```diff
@@ -708,22 +738,29 @@ export default abstract class Session {
         this.connection.stopPlaying();
 
         try {
-            let inputArgs = ["-ss", seekLocation.toString()];
+            let inputArgs = ["-ss", seekLocation!.toString()];
```

Comment:
Why is seekLocation nullable here?

Reply:
Sorry, this was a remnant of a previous implementation